src/subclasses/ContractInteractions.ts

```diff
@@ -487,14 +486,55 @@ class ContractInteractions {
 
   /**
    *
-   * @description swaps ETH for carbon pool tokens and uses them to retire carbon
-   * @notice this method may take up to even 1 minute to give a result
+   * @description retire  carbon credits using the lowest quality (oldest) TCO2 tokens available from the specified Toucan token pool by sending ERC20
+   * tokens (cUSD, USDC, WETH, WMATIC). All provided token is consumed for
+   * offsetting.
+   * @notice This method needs two different actions signed and may take up to even 1 minute to give a result
+   *  @dev When automatically redeeming pool tokens for the lowest quality
+   * TCO2s there are no fees and you receive exactly 1 TCO2 token for 1 pool
+   * token.
+   * @param swapToken portal for the token to swap into pool tokens (only accepts WETH, WMATIC and USDC)
```

Comment:
Again, `swapToken` variable name is confusing to me

Reply:
it's being used here as that is the token that will be swapped for pool tokens